package/yast2-bootloader.changes

```diff
@@ -1,3 +1,9 @@
+-------------------------------------------------------------------
+Wed May 29 12:02:02 UTC 2019 - David Diaz <dgonzalez@suse.com>
+
+- Improve the readability of a technical problem description,
+  splitting it in multiple lines.
```

Comment:
it need jira or bsc

Reply:
Discussed at IRC.

Basically, we do not have. And the intention was to include it along the changes sent by @schubi2 at #566.

Anyway, I am going to report the bug in order to use it also in next PRs for SLE-15

Thanks!